specification/applicationinsights/data-plane/LiveMetrics/preview/2024-04-01-preview/livemetrics.json

```diff
@@ -0,0 +1,1213 @@
+        "operationId": "Publish",
+        "description": "Publish live metrics to the Live Metrics service when there is an active subscription to the metrics.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/Azure.Core.Foundations.ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.ikey"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.apiKey"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.configurationEtag"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.transmissionTime"
+          },
+          {
+            "$ref": "#/parameters/PublishOperationRequestParameters.monitoringDataPoints"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/CollectionConfigurationInfo"
+            },
+            "headers": {
+              "x-ms-qps-subscribed": {
+                "type": "string",
+                "description": "Tells the client whether the input ikey is subscribed to."
+              },
+              "x-ms-qps-configuration-etag": {
+                "type": "string",
+                "description": "An encoded string that indicates whether the collection configuration is changed."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/ServiceError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Live Metrics client publishes live metrics": {
+            "$ref": "./examples/LiveMetrics_Publish.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AggregationType": {
+      "type": "string",
+      "description": "Aggregation type.",
+      "enum": [
+        "Avg",
+        "Sum",
+        "Min",
+        "Max"
+      ],
+      "x-ms-enum": {
+        "name": "AggregationType",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Avg",
+            "value": "Avg",
+            "description": "Average"
+          },
+          {
+            "name": "Sum",
+            "value": "Sum",
+            "description": "Sum"
+          },
+          {
+            "name": "Min",
+            "value": "Min",
+            "description": "Minimum"
+          },
+          {
+            "name": "Max",
+            "value": "Max",
+            "description": "Maximum"
+          }
+        ]
+      }
+    },
+    "CollectionConfigurationError": {
+      "type": "object",
+      "description": "Represents an error while SDK parses and applies an instance of CollectionConfigurationInfo.",
+      "properties": {
+        "CollectionConfigurationErrorType": {
```

Comment:
All JSON property names should be lower-camel-case and your service should use case-sensitive matching when deserializing these properties. I've only put this comment here but it applies to all the properties in the API.